app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, foreign_key: 'author_id'
+
```

Comment:
Please add a method in the Like model that updates the likes counter for a post.